spec/views/forms/show.html.erb_spec.rb

```diff
@@ -36,23 +36,21 @@
     expect(rendered).to have_selector(".app-task-list__summary", text: "You’ve completed 12 of 20 tasks.")
   end
 
-  context "when form state is draft", feature_groups: true do
```

Comment:
Is there any value to keeping this context? Why was it included in the first place?

Reply:
It adds context to the form status, which is draft by default.

I think we could go either way with this one. If we decide to add the context back, I think there are other tests that should be under it. We were erring on the side of less nesting.